src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -47,13 +48,16 @@
     private static final String MULTIPLOID_DATA_HG37 = largeFileTestDir + "gvcfs/HapMap5plex.ploidy10.b37.g.vcf";
     private static final String NA12878_HG37 = " src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testGVCFMode.gatk4.g.vcf";
     private static final String ARTIFICIAL_PHASED = getTestDataDir() + "/ArtificalPhasedData.1.g.vcf";
+    private static final String HG_00268_withSpaces = largeFileTestDir + "gvcfs/HG00268.spaceInSampleName.g.vcf";
     private static final List<String> LOCAL_GVCFS = Arrays.asList(HG_00096, HG_00268, NA_19625);
     private static final String GENOMICSDB_TEST_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/GenomicsDBImport/";
     private static final String COMBINEGVCFS_TEST_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/walkers/CombineGVCFs/";
 
     private static final String COMBINED = largeFileTestDir + "gvcfs/combined.gatk3.7.g.vcf.gz";
+    private static final String TRUNCATED_COMBINED = largeFileTestDir + "gvcfs/combined.gatk3.7.truncated.g.vcf";
```

Comment:
What does this name mean, `TRUNCATED_COMBINED`? How is this file truncated?

Reply:
Its a subsection of the longer file, I shortened it to be a much smaller portion of the larger file to cut down on size, especially since it was really just a test asserting that it didn't crash.

Reply:
"Truncated" implies that the file is incomplete or corrupt -- recommend using "SMALLER_INTERVAL_COMBINED" and "SMALLER_INTERVAL" instead.